File: Graph/Basic/Representation/AdjacencyMatrix/AdjacencyMatrix.java

import java.util.Arrays;

public class AdjacencyMatrix {

    static int[][] printGraph(int V, int[][] edges) {
        int[][] graph = new int[V][V];
        for (int[] edge : edges) {
            graph[edge[0]][edge[1]] = 1;
            graph[edge[1]][edge[0]] = 1;
        }
        return graph;
    }

    public static void main(String[] args) {
        int[][] edges = {{0, 1}, {0, 4}, {4, 1}, {4, 3}, {1, 3}, {1, 2}, {3, 2}};
        int[][] graph = printGraph(5, edges);
        for (int[] row : graph) {
            System.out.println(Arrays.toString(row));
        }
    }
}

File: Graph/Basic/Representation/AdjacencyMatrix/AdjacencyMatrix.cpp
#include<bits/stdc++.h>
using namespace std;
vector<vector<int>> printGraph(int V, vector<pair<int, int>> &edges) {
    vector<vector<int>> graph(V, vector<int>(V, 0));
    for(auto edge: edges) {
        graph[edge.first][edge.second] = 1;
        graph[edge.second][edge.first] = 1;
    }
    return graph;
}
int main(void) {
    vector<pair<int, int>> edges;
    edges.push_back(make_pair(0, 1));
    edges.push_back(make_pair(0, 4));
    edges.push_back(make_pair(4, 1));
    edges.push_back(make_pair(4, 3));
    edges.push_back(make_pair(1, 3));
    edges.push_back(make_pair(1, 2));
    edges.push_back(make_pair(3, 2));
    vector<vector<int>> graph = printGraph(5, edges);
    for(auto v: graph) {
        for(int i: v) {
            cout << i << " ";
        }
        cout << "\n";
    }
    return 0;
}